rockcraft.yaml

```diff
@@ -184,28 +197,13 @@ parts:
         group: 584792
         mode: "755"
 
-  trino-cli:
-    plugin: dump
-    source: https://repo.maven.apache.org/maven2/io/trino/trino-cli/418/trino-cli-418-executable.jar # yamllint disable-line
-    source-checksum: sha1/6cc447532d8722f78529257c13f8ab51e7481b15
-    source-type: file
-    organize:
-      trino-cli-418-executable.jar: trino
-    stage:
-      - trino
-    permissions:
-      - path: trino
-        owner: 584792
-        group: 584792
-        mode: "755"
-
   package-management:
     plugin: nil
     after: [trino, local-files, ranger-plugin]
     overlay-packages:
+      - openjdk-21-jdk-headless
```

Comment:
Question: why the move from stage to overlay?

Reply:
In overlay it was adding it in the path as well, so you could run java without specifying the path... but let me double check.

Reply:
yup confirmed